### PR TITLE
Some fixes to the core

### PR DESCRIPTION
**Pull request by f-person**

### CODE REVIEW

File: packages/dtogen/lib/application/json_dart_class_parser.dart [hunk 1]
```diff
@@ -113,15 +135,13 @@ class JsonDartClassParser extends IDartClassParser<Json> {
...
-      isNullable: isNullable,
-    );
+
+    return fieldType;
   }
 
   String _classNameFromKey(String key) {
-    var fieldTypeName = key.split("_").map((e) => e.firstCharToUpperCase()).join();
+    var fieldTypeName =
+        key.split("_").map((e) => e.firstCharToUpperCase()).join();
```
Comment (Maksimka101) [inline packages/dtogen/lib/application/json_dart_class_parser.dart]: Add an empty comment after the `key` for better formatting :)

File: packages/dtogen/lib/application/json_dart_class_parser.dart [hunk 2]
```diff
@@ -89,8 +92,27 @@ class JsonDartClassParser extends IDartClassParser<Json> {
...
+    final fieldName = key.toCamelCase();
+
+    return DartClassField(
+      name: fieldName,
+      typeName: fieldType,
+      isNullable: isNullable,
+    );
+  }
+
+  String _parseFieldType(
```
Comment (Maksimka101) [inline packages/dtogen/lib/application/json_dart_class_parser.dart:105]: Can you please create the enum for the field type?

File: packages/dtogen/lib/application/core/type_string_extensions.dart [hunk 3]
```diff
@@ -37,6 +39,14 @@ extension TypeStringExtensions on String {
     return replaceAll("Dto", "");
   }
 
+  String toCamelCase() {
+    return replaceAll(RegExp(r"(_|-)+"), " ")
+        .split(" ")
+        .map((str) => str.firstCharToUpperCase())
+        .join()
+        .firstCharToLowerCase();
```
Comment (Maksimka101) [inline packages/dtogen/lib/application/core/type_string_extensions.dart:47]: Does it makes sense? Previously you’ve called this method on the each part of this string

Comment (f-person) [reply packages/dtogen/lib/application/core/type_string_extensions.dart:47]: It's to**Lower**Case :)

**Pull request merged**